solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/UI/Extensions/ViewExtension.swift

```diff
@@ -0,0 +1,13 @@
+import SwiftUI
+
+#if DEBUG
+extension View {
+    /// Reshapes view into square in a specified color scheme. Use in previews.
+    func previewSquare(_ colorScheme: ColorScheme) -> some View {
```

Comment:
💯

Reply:
**[SUGGESTION]**
Consider this:
```suggestion
    func previewSquare(ofSize: CGFloat = Constants.preview_square_size, colorScheme: ColorScheme) -> some View {
```

Reply:
Looks like it can be useful in the future.
I just follow rule not to implement something that I'm not sure will be used in future. But if you are sure I will do it.

I see in suggestion you are letting coders to insert a custom number. Since we want to have numbers (constants) together, what do you think about this..

`func previewSquare(size: Constants.PreviewSquareSize = .medium, _ colorScheme: ColorScheme) -> some View {`

..in Constants file it would look like this and would be easy to add a new preview square size.
```
enum Constants {
    enum PreviewSquareSize: CGFloat {
        /// width = 400
        case medium = 400
    }
}
```